### ISSUE TITLE
Ambient sounds

### ISSUE THREAD

**Issue by MarkuBu** [D00Med/farlands#164]
I added my ambiance sounds mod, but currently it only supports sounds for ocean, forests, jungle, some underground ambiance and a simple wind sound for deserts.

Need suggestions for grassland, swamp, bamboo, tundra, icesheet, savanna and snowy grassland

Grassland could be some birds and humming bees. Swamp maybe frogs and buzzing mosquitoes. In the tundra you may hear a howling wolve.

And I think I will change the mod. I want to merge the single mods to one mod and I want to be able to reuse sounds for different biomes. Currently I have to add multiple copies of the same sound. I also want to check the biome

**Comment by tobyplowy** [comment]
> Grassland could be some birds and humming bees.

yes but the bees shuld not bee anoying (see what i did there not bee anoying)

> Grassland could be some birds and humming bees. Swamp maybe frogs and buzzing mosquitoes.

i like the frog sounds at the day but can you make them more intence at dawn (the moment when the sun is almost gone and the night begins) for the mosquitoes they shuld not be to anoying XD

>And I think I will change the mod. I want to merge the single mods to one mod and I want to be able to reuse sounds for different biomes. Currently I have to add multiple copies of the same sound. I also want to check the biome

yes that will be better

**Comment by MarkuBu** [comment]
Two bee or not two bee, that's the question xD

The problem is to find good (and free) ambient sounds. Suggestions are welcome

**Comment by tobyplowy** [comment]
> The problem is to find good (and free) ambient sounds. 

Well there's enough free sounds and there's enough good sounds 
but there's not enough good free sounds :/

> Suggestions are welcome

I don't really have any :/

> Two bee or not two bee, that's the question xD

I Rather have zero bee XD

**Comment by MarkuBu** [comment]
A lot of sounds are from http://www.orangefreesounds.com

e.g. a frog for the swamp http://www.orangefreesounds.com/frog-sound/
